x-pack/plugins/security/server/authentication/api_key.ts

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
__note:__ not related to this PR, but I guess at some point this file will be renamed to `api_keys.ts` anyway since we'll need to invalidate them through Security plugin or detect that the particular token has expired (similar to `tokens.ts` we have right now).

I guess at this stage we're not going to invalidate tokens through programmatic APIs, right? Also how alerting plugin deals with expired tokens? Do you have your own code that distinguish between expiration-related errors and others? If so, that'd make sense to move this logic here.

Reply:
So far we're not planning to make the API keys expire for alerting. In the future if we plan to do so, it would make sense to add logic here. I will rename the file to `api_keys.ts` anyway for consistency.